chain/store_test.go

```diff
@@ -207,7 +208,7 @@ func initSyncTest(t *testing.T,
 
 	fetcher := th.NewTestFetcher()
 	fetcher.AddSourceBlocks(calcGenBlk)
-	syncer := chain.NewSyncer(con, chainStore, messageStore, fetcher) // note we use same cst for on and offline for tests
+	syncer := chain.NewSyncer(con, chainStore, messageStore, fetcher, th.NewFakeSystemClock(time.Now())) // note we use same cst for on and offline for tests
```

Comment:
Don't use `time.Now()` in tests (or ever!), use a constant time value so tests are fully deterministic.